feature-configs/cn-ran-overlays/all-in-one/site.1.fqdn/networks/sriov-network-mh.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: mh-net-u-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam:  |
```

Comment:
I think this can be omitted completely. The driver type is vfio so it won't be needed.

Reply:
@josephdrichard

Reply:
I'll test if it still works with this removed.